core/backend/src/test/standalone/IModelWrite.test.ts

```diff
@@ -53,7 +54,7 @@ describe("IModelWriteTest", () => {
   });
   after(() => HubMock.shutdown());
 
-  it("Check busyTimeout option", async () => {
+  it.only("Check busyTimeout option", async () => {
```

Comment:
remove .only

Reply:
This test is failing for me locally but I can't figure out why. The failure doesn't seem to be related to the changes to Channels. I'll make the PR officially available for review so the tests are run in the cloud. At this point, I'm hoping it is just a fluke in my machine.

Reply:
i assume this has something to do with the change you removed from @khanaffan. I don't understand why you needed to do that (or why he made it necessary to pass `props` twice).

Reply:
The change from my side was certainly by mistake. I don't remember what exactly led me to change that line. Changing it back addressed the remaining test failure. Thanks!